test/safe/push.test.ts

```diff
@@ -25,7 +26,7 @@ describe("GitHub Actions", () => {
 				{
 					repository: event.payload.repository,
 					shas: [{ id: "test-commit-id", issueKeys: ["TEST-123"] }],
-					jiraHost: process.env.ATLASSIAN_URL,
```

Comment:
Tests depending on env vars 😬

Reply:
Why's that an issue if we're using the same env vars in our code?